components/match2/commons/match_group_input_util.lua

```diff
@@ -1346,8 +1343,19 @@ function MatchGroupInputUtil.standardProcessFfaMatch(match, Parser, mapProps)
 end
 
 ---@param opponents table[]
+---@param hasNoScores boolean?
 ---@return integer[]
-function MatchGroupInputUtil.calculatePlacementOfOpponents(opponents)
+function MatchGroupInputUtil.calculatePlacementOfOpponents(opponents, hasNoScores)
+	local manualPlacements = {}
+	Array.forEach(opponents, function(opponent, opponentIndex)
+		manualPlacements[opponentIndex] = opponent.placement
+	end)
+	-- if we have no scores or a placement for each opponent return early
+	-- as there is no need to determine what is already there or can not be determined
+	if hasNoScores or Table.size(manualPlacements) == #opponents then
```

Comment:
Why is the hasNoScore needed? Since this is only being called on finished records?